source/guides/guides/parallelization.md

```diff
@@ -236,7 +236,7 @@ Cypress currently uses the following CI environment variables to determine a CI
 Provider  | Environment Variable
 --|--
 AppVeyor  | `APPVEYOR_BUILD_NUMBER`
-Bamboo  | `BAMBOO_BUILD_NUMBER`
+Bamboo  | `bamboo_buildNumber`
```

Comment:
It seems we actually look for `bamboo.buildNumber` https://github.com/cypress-io/cypress/blob/develop/packages/server/lib/util/ci_provider.js#L81:L81

```suggestion
Bamboo  | `bamboo.buildNumber`
```